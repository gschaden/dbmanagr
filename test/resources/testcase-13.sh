skiline@localhost/skiline/api_key/
