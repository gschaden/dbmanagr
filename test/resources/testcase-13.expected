Id	Title	Subtitle	Autocomplete	
6	XML	09e779ae355b11e1960deb4f79cd2388	skiline@localhost/skiline/api_key/09e76932355b11e191dfaf54d84cf92e/
7	SMR	65c21f56176811e1b041005056b7001c	skiline@localhost/skiline/api_key/126a0fc4266011e1b7fe14feb5c97fba/
0	JFB API	15e30038abea11e19098000c29cccc10	skiline@localhost/skiline/api_key/290ae914abea11e1b40814feb5c97fba/
5	Android	2ae8ab4e56eb11e19fde000c297a6906	skiline@localhost/skiline/api_key/2ae8a93c56eb11e1a508000c297a6906/
5	BMW	f6d48dd2bd3711e2b3cd000c29cccc10	skiline@localhost/skiline/api_key/37f27048852e11e0b2f4005056b7001c/
0	Salzburger Nachrichten	2bbec3104a7811e293bf3c075440769f	skiline@localhost/skiline/api_key/3fbcb30454d711e28d8514feb5c97fba/
2	Mobile	e580b1d836a511e1af825ba4a2b345f7	skiline@localhost/skiline/api_key/4d1b343e37c911e1af2c14feb5c97fba/
8	Widgets	385b74f237a411e1aef41b6dc17e5743	skiline@localhost/skiline/api_key/583c30a437a411e1af6514feb5c97fba/
0	Portal	aba74274315f11e195008f30ce1bd58f	skiline@localhost/skiline/api_key/62ca1f0432b311e1bfb414feb5c97fba/
3	iPhone 1.0	bcb31638315f11e193061f49782cf09e	skiline@localhost/skiline/api_key/62d3da6232b311e1855014feb5c97fba/
1	Kiosk	86414cd0316111e1b28697be39619c7b	skiline@localhost/skiline/api_key/62d79c4c32b311e1bac314feb5c97fba/
0	Hikeline	ffea0d34c9cb11e2995da3700476952a	skiline@localhost/skiline/api_key/669ba310c9fb11e2aa1fd702d6148dbf/
0	Games on Ski	bc9070200b9711e298c7000c29c2ac8f	skiline@localhost/skiline/api_key/6a199ae4226511e2a79b14feb5c97fba/
0	SMR Youtube	64b921dc6e7e11e1ab13000c29c2ac8f	skiline@localhost/skiline/api_key/84f623646e7e11e1963014feb5c97fba/
0	Swisscom	aab20d8af2ac11e2b5dc000c29c2ac8f	skiline@localhost/skiline/api_key/a901651af2ae11e2aa179fb64f7ea60f/
4	iPhone 2.0	5dd826acf96111e0a5a6005056b7001c	skiline@localhost/skiline/api_key/b95a11d40eab11e1a19114feb5c97fba/
0	SMR Mobile	2b2c15c2589511e18f33b706d1ce50fe	skiline@localhost/skiline/api_key/ef33ee3c5bb411e18bd314feb5c97fba/

